### ISSUE TITLE
adb: more than one device/emulator

### ISSUE THREAD

**Issue by MattSkala** [ReactiveCircus/android-emulator-runner#288]
When running more instances of a runner and executing tests in parallel, the action fails while waiting for emulator to come online.

```
/Users/matt/Library/Android/sdk/platform-tools/adb shell getprop sys.boot_completed
  adb: more than one device/emulator
  The process '/Users/fitify/Library/Android/sdk/platform-tools/adb' failed with exit code 1
```

There should probably be `-s` argument to select the appropriate emulator to support cases when multiple emulators are running.

**Comment by mrk-han** [comment]
@MattSkala Thanks for the report! Would you mind sending us your `.yml` file, and show how you are sharding / parallelizing your tests?

I personally wouldn't try to run multiple emulators if using the standard GitHub runner infrastructure. But, perhaps if you are using a local or self-hosted runner than this could be useful to support. 

If not for testing, can you iterate your use-case?

**Comment by mrk-han** [comment]
Duplicate of https://github.com/ReactiveCircus/android-emulator-runner/issues/193

**Comment by mrk-han** [comment]
@MattSkala Gonna close this because duplicate, but we can communicate on #193
